<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCharacterWindow</class>
 <widget class="QWidget" name="SelectCharacterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SelectCharacterWindow</string>
  </property>
  <widget class="QComboBox" name="comboBoxCharacters">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelCharacters">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Персонаж</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLvL">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>49</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Уровень</string>
   </property>
  </widget>
  <widget class="QLabel" name="labClass">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Класс</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxStrenght">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelStrenght">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сила</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAgility">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ловкость</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntelligence">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Интелект</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAgility">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxIntelligence">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeaponSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Оружие</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMedicineSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Медицина</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMechanicSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Механика</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelElectronicsSkillName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Электроника</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelXp">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Опыт</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxElectronicsSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMechanicSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxMedicineSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxWeaponSkill">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXp">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelXpCh">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraitsName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Особенности</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTraits3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp1">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>210</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTraitsHelp3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>240</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoTalant">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Древо талантов</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить Изменения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPointsLeft">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Осталось очков характеристик</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
